roles/nodejs/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# nodejs version to install
+nodejs_version: "4.x"
```

Comment:
From the one side `4.x` looks confusing and I'm afraid this could be used in a wrong way. Additionally, not sure if it's a good idea to allow configuring this var to install untested by us node version.

From the other side some configuration like version is not bad.

Another idea is `nodejs_major_version: 4` and use `{{nodejs_major_version}}.x` in code.

Decide what looks best for you.